capellambse/model/diagram.py

```diff
@@ -302,8 +328,21 @@ def type(self) -> modeltypes.DiagramType:
             LOGGER.warning("Unknown diagram type %r", sc)
             return modeltypes.DiagramType.UNKNOWN
 
-    def _create_diagram(self) -> aird.Diagram:
-        return aird.parse_diagram(self._model._loader, self._element)
+    @property
+    def filters(self) -> t.MutableSet[str]:
+        """Return a set of currently activated filters on this diagram."""
+        return aird.ActiveFilters(self._model, self)
+
+    @filters.setter
+    def filters(self, filters: cabc.Iterable[str]) -> None:
+        active_filters = aird.ActiveFilters(self._model, self)
```

Comment:
For simplicity, we can delegate to the property getter.

```suggestion
        active_filters = self.filters
```